native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -2610,7 +2635,11 @@ static int proxy_cluster_post_config(apr_pool_t *p, apr_pool_t *plog,
             for (j = 0; j < balancer->workers->nelts; j++, workers++) {
                  proxy_worker *worker = *workers;
                  ap_log_error(APLOG_MARK, APLOG_EMERG, 0, s,
+#if MODULE_MAGIC_NUMBER_MAJOR == 20120211 && MODULE_MAGIC_NUMBER_MINOR >= 124
                               "%s BalancerMember are NOT supported %s", balancer->s->name, worker->s->name);
+#else
+                              "%s BalancerMember are NOT supported %s", balancer->s->name, worker->s->name_ex);
+#endif
```

Comment:
I believe there is an error – `name_ex` should be in the `if` branch, not in the `else` one.

Reply:
Yes it is, thanks.